tasks/badge.yaml

```diff
@@ -8,6 +10,9 @@ variables:
   - name: COMMON_ZARF
     description: Whether or not there is a common zarf.yaml file
     default: "true"
+  - name: PACKAGE_DIR
+    description: Base directory where packages are located, relative to the root of the repository
+    default: ./
```

Comment:
How would you feel about making this "." and then putting the slashes in where necessary? i think its easier to read.